Autocoders/Python/src/fprime_ac/generators/formatters.py

```diff
@@ -1413,7 +1413,7 @@ def argFilter(self, module, args, name="foo"):
                 new_args.append((id2, "U16", comment, "", "", [], []))
 
             elif type == "SCLK":
-                type = module[0].upper() + module[1:] + "ArgSclk"
+                _ = module[0].upper() + module[1:] + "ArgSclk"
```

Comment:
Can't we just delete this line?

Reply:
I made a lot of switches from variable -> _ but left the line in.  Given the overall quality of the code base, I cannot be sure that there aren't side-effects to function calls and what not.

Thus clean ups more substantial will require a full refactor.